packages/core/src/components/Box/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./react";
+
+export const foo = "bar";
```

Comment:
If there are changes in the packages but no changeset, CI fails. (expected)
https://github.com/kuma-ui/kuma-ui/actions/runs/5773663820/job/15649975959?pr=265